client/src/components/FlyingSiteMap.vue

```diff
@@ -11,7 +11,7 @@ import "leaflet/dist/leaflet.css";
 import L from "leaflet";
 import { GestureHandling } from "leaflet-gesture-handling";
 import "leaflet-gesture-handling/dist/leaflet-gesture-handling.css";
-import tileOptions from "@/config/mapbox.js";
+import { tileOptions } from "@/config/mapbox.js";
```

Comment:
Hier hätte ich auch noch beide Optionen gesehen, damit man evtl sieht "in welcher Schneise" der SP liegt.

Reply:
done